app/src/main/java/com/example/mylittlelibrary/data/Game.kt

```diff
@@ -0,0 +1,5 @@
+package com.example.mylittlelibrary.data
+
+import java.util.*
+
+data class Game(var id: Integer, var name : String, var systemName: String, var lendTo: String, var date: Date)
```

Comment:
This looks like an incomplete feature.
Are you using these anywhere?  I don't see them used.
If not in use, remove them.  Unused code is clutter, and bad documentation, which increases technical debt.
Only complete features, which meet the team's Definition of Done, should be merged to Main/Master.